param/reactive.py

```diff
@@ -1004,7 +1705,7 @@ def evaluate(*args, **kwargs):
             return bind(evaluate, *params)
         return evaluate
 
-    def _clone(self, operation=None, copy=False, **kwargs):
+    def _clone(self, operation=None, copy=False, **kwargs) -> 'rx':
```

Comment:
FYI. The `rx' type annotation here made a huge difference in VS Code interpretability of the code.